packages/expo-cli/src/commands/eject/ConfigValidation.ts

```diff
@@ -271,12 +271,36 @@ async function attemptModification(
 function logNoConfig() {
   Log.log(
     Log.chalk.yellow(
-      'No Expo config was found. Please create an Expo config (`app.config.js` or `app.json`) in your project root.'
+      `No Expo config was found. Please create an Expo config (${Log.chalk.bold`app.json`} or ${Log
+        .chalk.bold`app.config.js`}) in your project root.`
     )
   );
 }
 
-function warnAboutConfigAndThrow(type: string, message: string, edits: Partial<ExpoConfig>) {
+export async function attemptAddingPluginsAsync(
```

Comment:
I'm sure you have a good reason but I was confused about why this function is exported from the eject directory but used in the install command

Reply:
will refactor